backend/src/main/java/com/woowacourse/friendogly/footprint/controller/FootprintController.java

```diff
@@ -32,10 +31,7 @@ public ResponseEntity<Void> save(@RequestBody SaveFootprintRequest request) {
     }
 
     @GetMapping("/near")
-    public List<FindNearFootprintResponse> findNear(
-        @RequestParam double lat,
-        @RequestParam double lng
-    ) {
-        return footprintQueryService.findNear(new FindNearFootprintRequest(lat, lng));
+    public List<FindNearFootprintResponse> findNear(FindNearFootprintRequest request) {
```

Comment:
`@RequestParam` 명시해주는 것이 좋겠네요~!
```suggestion
    public List<FindNearFootprintResponse> findNear(@RequestParam FindNearFootprintRequest request) {
```